data/templates/Detector_CosmosDB.csx

```diff
@@ -0,0 +1,26 @@
+private static string GetQuery(OperationContext<AzureCosmosDB> cxt)
+{
+    return
+    $@"
+        let startTime = datetime({cxt.StartTime});
+        let endTime = datetime({cxt.EndTime});
+        cluster('Aks').database('AKSprod').<YOUR_TABLE_NAME>
```

Comment:
I think we should update those to some cosmosdb ones

Reply:
Thanks for catching this!